code/_onclick/hud/plane_master_controller.dm

```diff
@@ -13,6 +13,8 @@ INITIALIZE_IMMEDIATE(/atom/movable/plane_master_controller)
 	if(!istype(hud))
 		return
 	owner_hud = hud
+	if(istype(owner_hud))
+		return INITIALIZE_HINT_QDEL
```

Comment:
Can you explain why you added this specific if statement, and have it delete after init?

Reply:
That is cause of the possibility that this might get spawned during a specific unit test but without a owner hud this would cause numerous runtimes so if there is nothing like that no reason to keep it spawned

Reply:
And i just realized that this should be more like a not istype()